parts/Compound/CompoundListItem/DetailItem/index.js

```diff
@@ -30,7 +30,8 @@ const DetailItem = ({ item, userBoost, totalAPY }) => {
   const token1 = item.token1.address;
   const token2 = item.token2.address;
   const token3 = item.token3.address;
-  const dexTokenName = item.source == "Pangolin" ? "PNG" : item.source == "Trader Joe" ? "JOE" : "SNOB";
+  const dexTokenName = item.source == "Pangolin" ? "PNG" : item.source == "Trader Joe" ? "JOE" :
```

Comment:
can we format nested ternaries for better readability?

```
const item = cond1 ? res1
            : cond2 ? res2
            : cond3 ? res3
            : res4;
```

Reply:
agree